pages/common/pamtopng.md

```diff
@@ -0,0 +1,21 @@
+# pamtopng
+
+> Convert a PAM image to PNG.
+> See also: `pnmtopng`, `pngtopam`.
+> More information: <https://netpbm.sourceforge.net/doc/pamtopng.html>.
+
+- Convert the specified PAM image to PNG:
+
+`pamtopng {{path/to/image.pam}} > {{path/to/output.png}}`
+
+- Mark the specified color as transparent in the output image:
+
+`pamtopng -transparent {{color}} {{path/to/image.pam}} > {{path/to/output.png}}`
+
+- Include the text in the specified file as tEXt chunks in the output:
```

Comment:
Is the emphasis here required of text?

```suggestion
- Include the text in the specified file as text chunks in the output:
```

Reply:
As I understand form reading the documentation, this is a fixed term in the PNG format.